client/src/main/scripts/components/QuickShare.tsx

```diff
@@ -96,29 +105,30 @@ class QuickShare extends React.Component<QuickShareProps, QuickShareState> {
   };
   handleChangeComplete = () => {
     this.timerId = null;
-    const request: Share.ShareReq = this.createRequestObject();
+    const request: ShareReq = this.createRequestObject();
     this.createShareableURL(request);
     this.setState({ isCopiedToClipboard: false });
   };
   handleFacebookClick = () => {
-    const request: Share.ShareReq = this.createRequestObject();
+    const request: ShareReq = this.createRequestObject();
     this.logToServer(request);
     const url = this.createShareableURL(request);
     this.showFacebookDialog(url);
   };
   handleTwitterClick = () => {
-    const request: Share.ShareReq = this.createRequestObject();
+    const request: ShareReq = this.createRequestObject();
     this.logToServer(request);
     const url = this.createShareableURL(request);
     const twitterUrl = this.createTwitterUrl(url);
     window.open(twitterUrl, "_blank");
   };
-  logToServer = (request: Share.ShareReq) => {
-    postRequest(Routes.API_SHARE, request, () => {});
+  logToServer = (request: ShareReq) => {
+    // TODO: reqork saving
+    // postRequest(Routes.API_SHARE, request, () => {});
```

Comment:
This "logToServer" piece is the *only* part that we implemented of the server connection.  I think we're better off leaving it dead.